File: Programming Basics with Python/03 Conditional statements advanced/More exercises/02_Bike_Race.py
juniors = int(input())
seniors = int(input())
trace = input()
junior_tax = 0
senior_tax = 0
if trace == "trail":
    junior_tax = 5.5
    senior_tax = 7
elif trace == "cross-country":
    junior_tax = 8
    senior_tax = 9.5
    if juniors + seniors >= 50:
        junior_tax *= 0.75
        senior_tax *= 0.75
elif trace == "downhill":
    junior_tax = 12.25
    senior_tax = 13.75
elif trace == "road":
    junior_tax = 20
    senior_tax = 21.5
income = juniors * junior_tax + seniors * senior_tax
tax = income * 0.05
donation = income - tax
print(f"{donation:.2f}")

File: JS Basics/03 Nested Conditional Statements - More Exercises/02. Bike Race.js
function race(juniors, seniors, trace) {
    juniors = Number(juniors);
    seniors = Number(seniors);
    let taxJuniors = 0;
    let taxSeniors = 0;
    switch (trace) {
        case "trail":
            taxJuniors = 5.50 * juniors;
            taxSeniors = 7 * seniors;
            break;
        case "cross-country":
            taxJuniors = 8 * juniors;
            taxSeniors = 9.5 * seniors;
            if ((juniors + seniors) >= 50) {
                taxJuniors *= 0.75;
                taxSeniors *= 0.75;
            }
            break;
        case "downhill":
            taxJuniors = 12.25 * juniors;
            taxSeniors = 13.75 * seniors;
            break;
        case "road":
            taxJuniors = 20 * juniors;
            taxSeniors = 21.50 * seniors;
            break;
    }
    let donation = (taxSeniors + taxJuniors) * 0.95;
    console.log(donation.toFixed(2))

}
race(3, 40, "road")